### PR TITLE
Remove LSB_JOBID from env when using eclrun

### LINKED ISSUE #1099: MPI does not work when using eclrun
When running MPI runs with eclrun, simulations will not start

### PR DESCRIPTION
**Pull request by sondreso**
**Issue**
Resolves #1099 

**Approach**
MPIrun from Intel MPI will try to connect to LSF if it detects LSB_JOBID in the environment. Unset the LSB_JOBID when using `eclrun`

### CODE REVIEW

File: python/res/fm/ecl/ecl_config.py [hunk 1]
```diff
@@ -223,6 +223,12 @@ def _get_run_env(self, eclrun_env):
             eclrun_env.pop("PATH")
 
         env.update(eclrun_env)
+
+        # Having the LSB_JOBID set when executing MPI runs will
```
Comment (markusdregi) [inline python/res/fm/ecl/ecl_config.py]: Another option would be to add `LSB_JOBID: ~` in the configuration and drop environment variables mapping to `None`. Then you get to move it out of the code 🤷 I'm not entirely sure which is better 🤔

Comment (sondreso) [reply python/res/fm/ecl/ecl_config.py]: I thought about it, and now I

File: tests/res/fm/test_ecl_run_new.py [hunk 2]
```diff
@@ -22,7 +22,10 @@
 import yaml
 import pytest
 import re
+import json
+from unittest import mock
 
+import res
```
Comment (ManInFez) [inline tests/res/fm/test_ecl_run_new.py]: unused import?

Comment (sondreso) [reply tests/res/fm/test_ecl_run_new.py]: good catch

**Review by ManInFez — APPROVED** [review]
LGTM!

**Pull request merged**